d3a_api_client/aggregator.py

```diff
@@ -8,7 +8,8 @@
 from d3a_api_client.grid_fee_calculation import GridFeeCalculation
 from d3a_api_client.rest_device import RestDeviceClient
 from d3a_api_client.utils import get_uuid_from_area_name_in_tree_dict, buffer_grid_tree_info, \
-    create_area_name_uuid_mapping_from_tree_info, get_slot_completion_percentage_int_from_message
+    create_area_name_uuid_mapping_from_tree_info, get_slot_completion_percentage_int_from_message, \
+    log_bid_offer_confirmation
```

Comment:
It looks like imports can be refactored. If you do that, please favour using brackets `()` over `\` to group multi-line imports together.